eng/scripts/CodeChecks.ps1

```diff
@@ -101,7 +98,11 @@ try {
         if ($LastExitCode -ne 0) {
             $status = git status -s | Out-String
             $status = $status -replace "`n","`n    "
-            LogError "Generated code is not up to date. You may need to run eng\scripts\Update-Snippets.ps1 or sdk\storage\generate.ps1 or eng\scripts\Export-API.ps1"
+            LogError "Generated code is not up to date.`n" + `
+                "You may need to run rebase on the latest master, `n" + `
+                "run 'eng\scripts\Update-Snippets.ps1' if you modified sample snippets or other *.md files, `n" + `
+                "run 'eng\scripts\Export-API.ps1' if you changed public APIs `n" +
+                "run 'dotnet build /t:GenerateCode' to update the generated code."
```

Comment:
It might also be worth adding a link to more information in the Contributing.md. When the messages start to get long I find that to be a better approach.

Reply:
Contributing.md is not very easy to navigate through rn.

Reply:
Even if it isn't easy if you directly link to a section in it with more details it might make some of this easier to understand. 

At any rate it is just a suggestion to try and help enable folks to unblock themselves easier.

Reply:
https://github.com/Azure/azure-sdk-for-net/pull/19187